packages/scalecube-microservice/tests/implementations/Microservice.spec.ts

```diff
@@ -100,14 +145,40 @@ describe('Test the creation of Microservice', () => {
         services: [greetingService1, greetingService2],
       });
 
-      test('Test REQUEST_RESPONSE', () => {
+      test(`
+      Scenario: Invoke registered serviceS from proxy (RequestResponse)
+        Given   a service with definition and reference
+        And     definition and reference comply with each other
+        |service          |definition            |reference             |
+        |greetingService1 |hello: RequestResponse|hello: RequestResponse|
+        |                 |greet$: RequestStream |greet$: RequestStream |
+        |greetingService2 |hello: RequestResponse|hello: RequestResponse|
+        |                 |greet$: RequestStream |greet$: RequestStream |      
+        When    Microservice is created
+        And     Proxy is created from the Microservice
+        And     proxy invokes a RequestResponse method
+        Then    successful RequestResponse is received
+      `, () => {
         const greetingServiceProxy = ms.createProxy({
           serviceDefinition: greetingServiceDefinitionHello,
         });
         return expect(greetingServiceProxy.hello(defaultUser)).resolves.toEqual(`Hello ${defaultUser}`);
       });
 
-      test('Test observable', (done) => {
+      test(`
+      Scenario: Subscribe to a service from proxy (RequestStream)
+        Given   a service with definition and reference
+        And     definition and reference comply with each other
+        |service          |definition            |reference             |
+        |greetingService1 |hello: RequestResponse|hello: RequestResponse|
+        |                 |greet$: RequestStream |greet$: RequestStream |
+        |greetingService2 |hello: RequestResponse|hello: RequestResponse|
+        |                 |greet$: RequestStream |greet$: RequestStream |      
```

Comment:
Change to:
|service |definition |reference |
|greetingService1 |greet$: RequestStream |greet$: RequestStream |
|greetingService2 |hello: RequestResponse|hello: RequestResponse|